RequestManagementSystem/DB/RequestDB.py

```diff
@@ -517,7 +517,7 @@ def getRequestIDsList( self, statusList = None, limit = None, since = None, unti
 
       reqQuery = reqQuery.order_by( Request._LastUpdate )\
                          .limit( limit )
-      requestIDs = [reqIDTuple[0] for reqIDTuple in reqQuery.all()]
+      requestIDs = [ tuple( reqIDTuple ) for reqIDTuple in reqQuery.all() ]
```

Comment:
Everywhere I saw it used. Since RequestNames use was replaced by RequestIDs, for example:
change of getRequestNames() -> getRequestIDsList(). The getRequestNames was also returning tuples. So, it was always like that.